CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This file is used to list changes made in each version of the apache2 cookbook.
 
 ## Unreleased
 
+- Fix cache_dir permission so that modules can write in their cache_dir/<module>/ storage space
```

Comment:
With the `<>` it doesn't render the word module as it's interpreted as html
```suggestion
- Fix cache_dir permission so that modules can write in their cache_dir/module/ storage space
```